src/main/java/budgetbuddy/model/UserPrefs.java

```diff
@@ -55,11 +57,20 @@ public Path getLoansFilePath() {
         return loansFilePath;
     }
 
+    public Path getAccountsFilePath() {
+        return accountsFilePath;
+    }
+
     public void setLoansFilePath(Path loansFilePath) {
         requireNonNull(loansFilePath);
         this.loansFilePath = loansFilePath;
     }
 
+    public void setAccountsFilePath(Path accountsFilePath) {
+        requireNonNull(accountsFilePath);
+        this.accountsFilePath = accountsFilePath;
+    }
+
     public Path getRuleFilePath() {
         return ruleFilePath;
     }
```

Comment:
This whole section was written as a "getGuiFP setGuiFP getLoanFP setLoanFP getRuleFP setRuleFP".

I think you might have misread this part and just inserted your getter and setter in the wrong place

Now it's "getGuiFP setGuiFP getLoanFP getAccountFP setLoanFP setAccountFP getRuleFP setRuleFP"

Perhaps you might want to follow the original flow

Reply:
okay.